tests/tracking/test_tracking.py

```diff
@@ -534,6 +541,13 @@ def test_log_batch_validates_entity_names_and_values():
             tracking.MlflowClient().log_batch(run_id, tags=tags)
         assert e.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
 
+        metrics = [Metric(key=None, value=42.0, timestamp=4, step=1)]
```

Comment:
Added this after splitting out the `None` check on the key name. The original exception message was a little confusing for this case.